packages/components/src/components/carousel/carousel.stories.ts

```diff
@@ -0,0 +1,234 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import '../../solid-components';
+import { html } from 'lit';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+
+const { argTypes, parameters } = storybookDefaults('sd-carousel');
+const { overrideArgs } = storybookHelpers('sd-carousel');
+const { generateTemplate } = storybookTemplate('sd-carousel');
+
+export default {
+  title: 'Components/sd-carousel',
+  component: 'sd-carousel',
+  args: overrideArgs([
+    {
+      type: 'slot',
+      name: 'default',
+      value: `
+        <sd-carousel-item> <slot-comp></slot-comp> </sd-carousel-item>
+        <sd-carousel-item> <slot-comp></slot-comp> </sd-carousel-item>
+        <sd-carousel-item> <slot-comp></slot-comp> </sd-carousel-item>
+        <sd-carousel-item> <slot-comp></slot-comp> </sd-carousel-item>
+        <sd-carousel-item> <slot-comp></slot-comp> </sd-carousel-item>`
```

Comment:
As you're currently working on the slot task – could you change this to BEM-styled divs, too?

Reply:
When you're done: approved.